include/rc_export.h

```diff
@@ -0,0 +1,69 @@
+#ifndef RC_EXPORT_H
+#define RC_EXPORT_H
+
+/* These macros control how callbacks and public functions are defined */
+
+/* RC_SHARED should be defined when building rcheevos as a shared library (e.g. dll/dylib/so) */
+/* RC_STATIC should be defined when building rcheevos as a static library */
+/* BUILDING_RC should be defined when building rcheevos itself, and should not be defined for external code */
+
+/* TODO: BUILDING_RC currently only has an effect for RC_SHARED (this is abused for test code), perhaps when it is undefined internal definitions should be hidden? */
+
+#if !defined(RC_SHARED) && !defined(RC_STATIC)
+  #error RC_SHARED or RC_STATIC must be defined
+#endif
```

Comment:
Rather than error out, I think it should default to RC_STATIC. You're probably the only consumer currently that doesn't just embed the c files directly in their executable, and not having a default would be a breaking change.